src/components/AddTodo.tsx

```diff
@@ -0,0 +1,156 @@
+
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addTodo } from '../features/todo/todoSlice';
+import './AddTodo.css'; // Import the CSS file
+import { useForm, SubmitHandler } from 'react-hook-form';
+
+type Inputs = {
+  exampleRequired: string;
+};
```

Comment:
Please move all types to `types/data.d.ts` file. Explore global type declaration.